app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -73,17 +75,20 @@ const val LAYER_ID = "0128"
 fun MapScreen(
     navigationActions: NavigationActions,
     parkingViewModel: ParkingViewModel,
+    userViewModel: UserViewModel,
     mapViewModel: MapViewModel,
     zoomState: MutableState<Double> = remember { mutableDoubleStateOf(defaultZoom) }
 ) {
 
   val listOfParkings by parkingViewModel.rectParkings.collectAsState()
+  val enableParkingAddition by userViewModel.isSignedIn.collectAsState(false)
+
   val mapViewportState = MapConfig.createMapViewPortStateFromViewModel(mapViewModel)
   var removeViewAnnotation = remember { true }
-  var cancelables = remember<Cancelable> { Cancelable({}) }
+  var cancelables = remember { Cancelable {} }
   var listener = remember<MapIdleCallback?> { null }
   var pointAnnotationManager by remember { mutableStateOf<PointAnnotationManager?>(null) }
-  val selectedParking = parkingViewModel.selectedParking.collectAsState().value
+  val selectedParking by parkingViewModel.selectedParking.collectAsState()
```

Comment:
@LouisJamesVasseur @Pijukai  😉

Reply:
@sean-prz That's just to simplify but at the end it doesn't change anything 😉